docs/components/clipboard-button.tsx

```diff
@@ -0,0 +1,36 @@
+import { createSignal, Show } from "solid-js";
+import { ClipboardIcon } from "./icons/clipboard-icon";
+
+interface CopyToClipboardProps {
+  class?: string;
+  manager: string;
+  command: string;
+}
+
+export function CopyToClipboard(props: CopyToClipboardProps) {
+  const copyText = () => `${props.manager} ${props.command}`;
+
+  const [isCopied, setIsCopied] = createSignal(false);
+
+  const copyToClipboard = async () => {
+    try {
+      await navigator.clipboard.writeText(copyText());
```

Comment:
We have a utility for this :) https://github.com/solidjs-community/solid-primitives/tree/main/packages/clipboard#readme

But either way this looks fine the way it is.